include/mbedtls/cipher.h

```diff
@@ -640,8 +684,8 @@ int mbedtls_cipher_reset( mbedtls_cipher_context_t *ctx );
  *                      buffer of at least \p ad_len Bytes.
  * \param ad_len        the Length of \p ad Bytes.
  *
- * \return              \c 0 on success.
- * \return              A specific error code on failure.
+ * \return            \c 0 on success.
```

Comment:
Minor: The indentation isn't correct here.

Reply:
If you fix this, could you fix the documentation of `ad_len` along the way? It should be "The length of \p ad in Bytes."

Reply:
Done.